functions-methods.js

```diff
@@ -33,4 +65,44 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity (email) {
+     if (email.includes("@") && !email.includes(",") && email.slice(-1) !== ".") {
+         console.log( email + " is valid");
+
+     }else if (email.slice(-1) === "." || email.includes(",") || !email.includes("@")) {
+         console.log("email not valid");
+     }
+
+    switch (true) {
+        case email.includes("@") && !email.includes(",") && email.slice(-1) !== ".":
+                console.log("geeft true - want @ en punt op de juiste plek");
+            break;
+
+        case email.slice(-1) === ".":
+                console.log("geeft false - want de punt mag niet als laatst");
+            break;
+
+        case email.includes(","):
+                console.log("geeft false - want er staat een komma in");
+            break;
+
+        case !email.includes("@"):
+                console.log("geeft false - want geen @");
+            break;
+
+        default:
+            console.log("not valid email")
+    }
```

Comment:
Een statische waarde in een switch is niet echt hoe het hoort. Eigenlijk ben je dan een if-else structuur aan het toepassen in een switch, en daarom zie ik dan liever een échte if-else structuur
```